scripts/finalize-template.py

```diff
@@ -11,43 +11,40 @@ def get_parser():
         "Fills out argument templates, then destroys, saves the new files,"
```

Comment:
This source file got "blackened" and thus the changes are pretty noisy. The key here is in (new) lines 41-47 which define an `app_name_hyphen` and `app_name_underscore` so things can be more explicitly provided in a way that works regardless of what you named the repository